src/app/Library/Widget.php

```diff
@@ -48,6 +48,17 @@ public static function add($attributes = null)
         return new static($attributes);
     }
 
+    /**
+     * Return the widget attribute value or null when attribute does not exist.
+     *
+     * @param  string  $attribute
+     * @return mixed
+     */
+    public function getAttribute(string $attribute)
+    {
+        return $this->attributes[$attribute] ?? null;
+    }
+
```

Comment:
I noticed `getAttribute()` might returning NULL in two cases:
- (1) when the attribute is `null`
- (2) when the attribute doesn't exist

It's the same problem we had with the `request()`, remember? No way to differentiate if an input is _actually_ null or just doesn't exit.

I think we'll have that confusion pretty soon, so what do you think about also adding:

```suggestion

    /**
     * Check if the widget has an attribute.
     *
     * @param  string  $attribute
     * @return bool
     */
    public function hasAttribute(string $attribute)
    {
        return array_key_exists($attribute, $this->attributes);
    }
```

PS. Did NOT test the code above - you please do if you agree. But it should work, right?

Reply:
Totally, just updated PR!